splice_machine/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/TernaryOperationIT.java

```diff
@@ -284,8 +287,27 @@ public void testSubString() throws Exception {
             sCell2 = rs.getString(2);
         }
         Assert.assertEquals("Wrong substr result", sCell1, sCell2 );
+    }
 
+    @Test
+    public void testLeft() throws Exception {
+        ResultSet rs;
 
+        String sql =
+            "select left(name, cut, 'short') from (" +
+            "values ('hello world', 5), ('hey dude', 20)" +
+            ") tb(name, cut);";
+
+        List<String> expected = new ArrayList<String>(2);
+        expected.add("hello");
+        expected.add("hey dudeshortshortsh");
+
+        rs = methodWatcher.executeQuery(sql);
+        int i = 0;
+        while (rs.next()) {
+            Assert.assertEquals(expected.get(i), rs.getInt(1));
```

Comment:
Haven't executed it myself, and am curious about `rs.getInt(1)`. How can `int` be equal to `String` in this (test) case?

Reply:
Thanks, good catch! Fixed. I was testing with length(), then changing to string compare. My concern is why this passed the test. For now, the fix should be correct. Please review.

Reply:
If you're at it, I'd rather use the array of strings `expected` as the source of truth, i.e. I'd iterate over the expected elements not `rs.next()` in `while`.